docs/content/how-to/aws/create-aws-hosted-cluster-arm-workers.md

```diff
@@ -1,10 +1,66 @@
 ---
-title: Create ARM NodePools on AWS Hosted Cluster
+title: Create Arm NodePools on AWS Hosted Clusters
 ---
 
-# Create ARM NodePools on AWS Hosted Cluster
+# Create Arm NodePools on AWS Hosted Clusters
 
-## Creating a Hosted Cluster with ARM NodePool
+The `arch` flag was added to the NodePool Spec in OCP 4.14. The `arch` flag sets the preferred processor architecture for the NodePool (currently only supported on AWS).
```

Comment:
s/flag/field/

Reply:
also, is it really preferred? or required to be the arch requested?

Reply:
Required really, I `s/preferred/required`.